odpc.client/src/features/waardelijst/components/CheckboxList.vue

```diff
@@ -1,31 +1,38 @@
 <template>
-  <details>
-    <summary>{{ title }}</summary>
+  <details :data-required-group="required ? 'checkbox' : null">
+    <summary>{{ title }} {{ required ? "*" : "" }}</summary>
+
+    <p v-if="required" class="error" :id="`description-${getCurrentInstance()?.uid}`">
+      Kies minimaal één optie.
+    </p>
 
     <div class="checkbox check-all">
       <label
-        ><input type="checkbox" @click="toggleAll" :checked="allSelected" /> selecteer alles
+        ><input
+          type="checkbox"
+          @click="toggleAll"
+          :checked="allSelected"
+          :aria-describedby="`description-${getCurrentInstance()?.uid}`"
+        />
+        selecteer alles
       </label>
     </div>
 
-    <div class="checkbox" v-for="({ id, name }, key) in options" :key="key">
-      <label><input type="checkbox" :value="id" v-model="model" /> {{ name }} </label>
+    <div class="checkbox" v-for="({ uuid, naam }, key) in options" :key="key">
+      <label><input type="checkbox" :value="uuid" v-model="model" />{{ naam }}</label>
```

Comment:
is hier ook een aria-describedby naar de description nodig?

Reply:
Goeie vraag. Ik had alleen op de eerste gezet omdat daar de focus op komt bij foutmelding. Kan hier geen eenduidig antwoord op vinden, maar heb nu ook op andere checkboxes gezet.